db/migrate/20211203194905_add_user_ref_to_like.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLike < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] Again, the reference name should be `author`. Please refer the above similar comment for the other migration. :pray: